helm/templates/config-service-config.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Values.configServiceConfig.name }}
+  labels:
+    release: {{ .Release.Name }}
+data:
+  application.conf: |-
+    document.store {
```

Comment:
are we not exposing the other properties of the conf? Specifically, the ports, since those are already parameterized elsewhere.

Reply:
We need to do it. Updated.